data/test/index.js

```diff
@@ -59,3 +59,26 @@ describe( 'query', () => {
 		expect( tree ).toMatchSnapshot();
 	} );
 } );
+
+describe( 'subscribe', () => {
+	it( 'registers multiple selectors to the public API', () => {
+		let data = null;
+		const store = registerReducer( 'myAwesomeReducer', ( state = 0 ) => state + 1 );
+		registerSelectors( 'myAwesomeReducer', {
+			globalSelector: ( state ) => state,
+		} );
+		const unsubscribe = subscribe( () => {
+			data = select( 'myAwesomeReducer', 'globalSelector' );
+		} );
+		const action = { type: 'dummy' };
+		store.dispatch( action ); // increment the data by => data = 2
+		store.dispatch( action ); // increment the data by => data = 3
+
```

Comment:
I would prefer to see `expect` calls here instead of comments. Can we also rename `data` to `counter` or `inrecementedValue`?
We can also rename `unsubscribe` to `unsubscribeFromIncrementing` :)

It might be only me, but I don't like to see inline comments in tests.